cs-property-service/src/main/java/org/egov/cpt/validator/PropertyValidator.java

```diff
@@ -274,14 +275,11 @@ private void validateTransitNumber(PropertyRequest request, Map<String, String>
 						errorMap);
 				compareIds(propertySearch.getTransitNumber(),
 						property.getPropertyDetails().getAddress().getTransitNumber(), errorMap);
-				propertySearch.getOwners().forEach(searchOwner -> {
-					property.getOwners().forEach(owner -> {
-						compareIds(searchOwner.getId(), owner.getId(), errorMap);
-						compareIds(propertySearch.getId(), owner.getProperty().getId(), errorMap);
-						compareIds(searchOwner.getOwnerDetails().getId(), owner.getOwnerDetails().getId(), errorMap);
-						compareIds(propertySearch.getId(), owner.getOwnerDetails().getPropertyId(), errorMap);
-						compareIds(searchOwner.getId(), owner.getOwnerDetails().getOwnerId(), errorMap);
-					});
+				List<String> oIdList=propertySearch.getOwners().stream().map(Owner::getId).collect(Collectors.toList());
+				property.getOwners().forEach(owner -> {
+					if(!oIdList.contains(owner.getId())){
+						errorMap.put("INVALID ID", "ID is not valid");

```

Comment:
Can you make this more descriptive?
`Existing user with id '2626-22ywywy-4wy42-4616-31ab' is missing during update. All the owners should be present during the update API`